frontend/app/routes/index.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+import jQuery from 'jquery';
+
+export default Ember.Route.extend({
+  model: function() {
+    return this.store.find('story', {
+      landing: true
+    });
+  },
+
+ setupController: function(controller, model) {
+    this._super(controller, model);
+    Ember.run.scheduleOnce('afterRender', () => {
+      jQuery.mark.jump({
+        offset: -120
```

Comment:
Where does this magic number come from, what is `jQuery.mark.jump`, and should it have a comment explaining?